content/ecosystem/bridges/discord/bridges.toml

```diff
@@ -12,16 +12,16 @@ featured = true
 privilege.platform = "Admin" # Free text
 privilege.matrix = "Homeserver Admin" # Any of Homeserver Admin, Room Admin, None
 [bridges.supports]
-dm = true
+dm = false
 channels = true
 formatted_text = true
 message_media = true
-replies = true
+replies = false
 mentions = true
```

Comment:
probably not for too long https://github.com/matrix-org/matrix-spec-proposals/pull/4210